src/BUILD.bazel

```diff
@@ -18,6 +18,13 @@ genrule(
     cmd = "cp $< $@",
 )
 
+genrule(
```

Comment:
we should clean this up - can be in a later PR. the ts_devserver should have an option that allows us to serve additional rootdirs, I think this is already present but maybe not wired up?